angular/projects/spark-angular/src/lib/components/sprk-masthead/components/sprk-masthead-accordion/sprk-masthead-accordion.component.ts

```diff
@@ -18,6 +22,12 @@ export class SprkMastheadAccordionComponent {
    */
   @Input()
   additionalClasses: string;
+  /**
+   * Value supplied will be used
+   * as an `aria-label` on the component.
+   */
+  @Input()
+  ariaLabel: string;
```

Comment:
I added an aria label input here because the accordion is wrapped in a nav element.